digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/MunicipalityApi.scala

```diff
@@ -890,15 +890,27 @@ class MunicipalityApi(val onOffLinearAssetService: OnOffLinearAssetService,
     if(assetService.getMunicipalityById(municipalityCode).isEmpty)
```

Comment:
Is municipalityCode needed at all? Delete is done by assetId, and it was removed from the PUT and POST